### ISSUE TITLE
would be nice if extensions.gnome.org is updated

### ISSUE THREAD

**Issue by askmrsinh** [orangeshirt/gnome-shell-extension-touchpad-indicator#55]
The official GNOME Extensions page at https://extensions.gnome.org/extension/131/touchpad-indicator/ is outdated.

**Comment by orangeshirt** [comment]
Sorry - I stop the development of this extension 2 years ago, because I'm not longer use Gnome. That's why I don't have the possibillity to test the extension. And I don't want to upload an extension without testing.
But you or someone else could fork or download the extension from here and upload it to extensions.gnome.org

**Comment by user501254** [comment]
Understood, I will fork the extension in the coming week and update on extensions.gnome.org since there's no other similar working alternative.
This is an useful extension. Thank you.

**Comment by askmrsinh** [comment]
I gave it a shot, but came to the realization that it would be almost impossible to get through the reviewers. To have 2-3 extensions which do the same thing (even if one is outdated) is not good for the ecosystem.

**Comment by gearhead** [comment]
This is pretty odd, IMO. The only version which is 'on the web interface'
no longer works or at least does not for me. The original author will not
update it as he cannot test it to verify function. How can we pass the
baton, so to speak, so the 'official version' will still work? How can we
have Ashesh take over the extension so that it gets updated. As it is, the
version I have installed will not release the touchpad when the mouse is
plugged in, I get both. It used to work perfectly.

**Comment by askmrsinh** [comment]
Actually, there are two:
1. [Touchpad Indicator](https://extensions.gnome.org/extension/131/touchpad-indicator/) (ie. this project)
2. ~~[Toggle Touchpad](https://extensions.gnome.org/extension/935/toggle-touchpad/)~~

The second one is more frequently updated, last update being 26 days ago. I haven't tried it out but reading the reviews it feels like it works well.
Also, I should point out that even if I fork the project I will only be able to do the maintenance and won't be able to actively develop it due to my limited knowledge in JS.

**Comment by gearhead** [comment]
Neither works for me under Ubuntu 14.04. Used to work  perfectly

**Comment by askmrsinh** [comment]
@orangeshirt  
I have made a fork of your extension at [Auto Touchpad](https://user501254.github.io/gnome-shell-extension-auto-touchpad/) which I uploaded to the GNOME Shell Extensions website. ([It's awaiting approval](https://extensions.gnome.org/review/5436)).
However, it seems like the ownership of the original extension (Touchpad Indicator) on the website can be transferred to anyone which would be much better than having to upload a fork with different name.

Hence, I request you to send an email with the title "touchpad-indicator@orangeshirt ownership transfer to user501254" to mengzhuo1203@gmail.com with your registered email-id. I can then upload new versions of the extension with same name on the web interface and continue development.
